src/sprint-mission4/03-domain/service/user.service.js

```diff
@@ -0,0 +1,156 @@
+import { Exception } from "../../common/const/exception.js";
+import { User } from "../entity/user.js";
+
+export class UserService {
+  #repos;
+  #managers;
+
+  constructor(repos, managers) {
+    this.#repos = repos;
+    this.#managers = managers;
+  }
+
+  signUpUser = async ({ email, nickname, image, password }) => {
+    const foundUser = await this.#repos.user.findUserByEmail(email);
+
+    if (foundUser) {
+      throw new Exception("USER_EXIST");
+    }
+
+    const hashPassword = await this.#managers.hash.hashingPassword(password);
+
+    const newUser = User.forCreate({
+      email,
+      nickname,
+      image,
+      password: hashPassword,
+    });
+    const createdUser = await this.#repos.user.create(newUser);
+
+    return createdUser;
+  };
+  getUser = async ({ id }) => {
+    const foundUser = await this.#repos.user.findUserById(id);
+    if (!foundUser) {
+      throw new Exception("USER_NOT_EXSIST");
+    }
+    return foundUser;
+  };
+  getUserProducts = async ({ id, offset, limit, sort }) => {
+    const orderBy =
```

Comment:
```suggestion
  };
  
  getUser = async ({ id }) => {
    const foundUser = await this.#repos.user.findUserById(id);
    if (!foundUser) {
      throw new Exception("USER_NOT_EXSIST");
    }
    return foundUser;
  };
  
  getUserProducts = async ({ id, offset, limit, sort }) => {
    const orderBy =
```

함수 정의 사이 아시 적절하게 공백을 주어서 가독성을 높여주면 좋을 것 같습니다.

Reply:
확인했습니다!